### DISCUSSION TITLE
How to track newly created Termux packages ?

### DISCUSSION THREAD

**Discussion by maaaaz** [termux/termux-packages#14959] [Q&A]
Hello there,

How would it properly possible to track *newly created* packages on a changelog page ?

By *"newly created"*, I mean these two possibilities, but my question is really about the first one :
* **Brand new packages**: like the result of this page https://github.com/termux/termux-packages/pulls?q=is%3Apr+is%3Aclosed+new+package

* **New versions of already created packages**: like the result of this page https://github.com/termux/termux-packages/pulls?q=is%3Apr+is%3Aclosed+bump

Is there **any other ways** to track these 2 changelogs ?

Best regards, and keep rocking !

**Comment by 2096779623** [comment]
https://docs.github.com/en/rest/search?apiVersion=2022-11-28#search-issues-and-pull-requests

**Comment by sylirre** [comment]
Clone git repo and then track ADD changes of build.sh files:
```
git whatchanged --diff-filter=A packages/*/build.sh
```

**Comment by maaaaz** [comment]
Ok thank you for these answers !
Any plan to publish a specific `new-packages-changelog.txt` file somewhere ?

**Comment by SunPodder** [comment]
A python apt wrapper `nala` does this thing. It shows you the new and updated packages when you update your package list (`nala update` equivalent to `apt update`). You may look into its source
